src/ucs/config/parser.c

```diff
@@ -379,6 +379,145 @@ int ucs_config_sprintf_time(char *buf, size_t max, void *src, const void *arg)
     return 1;
 }
 
+int ucs_config_sscanf_speed(const char *buf, void *dest, const void *arg)
+{
+    size_t *dst     = (size_t*)dest;
+    char    str[16] = {0};
+    int     offset  = 0;
+    size_t  divider;
+    size_t  units;
+    double  value;
+    int     num_fields;
+
+    num_fields = sscanf(buf, "%lf%16s", &value, str);
+    if (num_fields < 2) {
+        return 0; /* bytes or bits? */
+    }
+
+    ucs_assert(num_fields == 2);
+
+    switch (str[offset]) {
+    case 'T':
+        units = UCS_TBYTE;
```

Comment:
can we unite common code with ucs_config_sscanf_memunits() ?

Reply:
if we add common function: `double ucs_config_parser_quantity_prefix_value(char prefix)`
input- **upper case** character - B,K,M,G,T
output - 1024**n

it can be used in ucs_config_sscanf_memunits() which is case-insensive with "toupper" and here

Reply:
what about bits?

Reply:
we can check 'B'/'b' before and after calling this function

Reply:
ok, done